### PR TITLE
add several items in zh-Hans and fixed some other minor bugs.

### PR DESCRIPTION
**Pull request by yangfei4913438**
### What does it do?

- Add several translations for zh-Hans.  
- Fix some plain characters that don't have translated text.
- other minor bugs...

### Why is it needed?

- Current pkg lack of these items.
- The tags that are displayed should use a multilingual scheme.

### How to test it?

eg:

```json
{
   "app.components.LeftMenu.general": "通用",
    "app.components.LeftMenu.plugins": "插件",
    "Settings.profile.form.section.experience.mode.label": "界面风格",
    "Settings.profile.form.section.experience.mode.hint": "将会用所选择的风格显示你的界面",
    "Settings.profile.form.section.experience.mode.option-label": "{name} 界面",
    "light": "浅色",
    "dark": "深色",
    "global.change-password": "修改密码",
    "global.password": "密码",
    "app.component.table.duplicate": "创建 {target} 的副本",
    "content-manager.components.DynamicTable.row-line": "第 {number} 条记录",
    "content-manager.select.currently.selected": "已选中 {count} 项",
    "global.delete": "删除",
    "global.description": "描述",
    "global.cancel": "取消",
    "global.details": "详情",
    "global.finish": "完成",
    "header.actions.add-assets": "添加新素材",
    "form.input.description.file-alt": "当此素材无法显示时，会显示该文本。",
    "input.placeholder": "点击选择一个素材，或者拖拽文件至区域",
    "input.placeholder.icon": "在此区域内放置素材",
    "list.assets.empty": "上传你的第一个素材",
    "list.assets.selected": "{number, plural, one {# 个素材} other {# 个素材}}准备上传",
    "mediaLibraryInput.actions.nextSlide": "下一张幻灯片",
    "mediaLibraryInput.actions.previousSlide": "上一张幻灯片",
    "mediaLibraryInput.placeholder": "点击添加素材，或拖放一个素材到当前区域",
    "mediaLibraryInput.slideCount": "共 {m} 张，第 {n} 张",
    "modal.upload-list.footer.button": "上传 {number, plural, one {# 个素材} other {# 个素材}}到媒体库",
    "global.plugins.content-manager": "内容管理",
    "global.plugins.content-manager.description": "在你的数据库里，快速的查看、编辑和删除数据。",
    "global.plugins.content-type-builder": "模型构建器",
    "global.plugins.content-type-builder.description": "对你的 API 数据结构进行建模。在短短的一分钟内创建新的字段和关系。这些文件会在你的项目中自动创建和更新。",
    "global.plugins.email": "电子邮件",
    "global.plugins.email.description": "配置你的应用程序来发送电子邮件。",
    "global.plugins.upload": "媒体库",
    "global.plugins.upload.description": "媒体文件管理。",
    "global.plugins.graphql": "GraphQL",
    "global.plugins.graphql.description": "添加具有默认 API 方法的 GraphQL 端点",
    "global.plugins.documentation": "文档",
    "global.plugins.documentation.description": "创建一个 OpenAPI 文档，用 SWAGGER UI 来可视化你的 API。",
    "global.plugins.i18n": "国际化",
    "global.plugins.i18n.description": "这个插件可以从管理面板和 API 中创建、阅读和更新不同语言的内容。",
    "global.plugins.sentry": "Sentry",
    "global.plugins.sentry.description": "将 Strapi 错误事件发送给 Sentry。",
    "global.plugins.users-permissions": "角色和权限",
    "global.plugins.users-permissions.description": "用一个基于 JWT 的完整认证过程来保护你的API。这个插件还带有一个 ACL 策略，允许你管理用户组之间的权限。",
     "global.roles": "角色列表",
     "global.users": "用户列表",
     "link": "链接",
     "Settings.email.plugin.button.test-email": "发送测试邮件",
     "Settings.email.plugin.label.defaultFrom": "默认发件地址",
     "Settings.email.plugin.label.defaultReplyTo": "默认回复地址",
     "Settings.email.plugin.label.provider": "电子邮件供应商",
     "Settings.email.plugin.title.test": "测试电子邮件发送",
     "Settings.email.plugin.label.testAddress": "收件地址",
     "Settings.email.plugin.title": "配置",
     "Settings.email.plugin.subTitle": "测试电子邮件插件的配置",
     "Settings.email.plugin.title.config": "配置",
     "Settings.email.plugin.text.configuration": "通过 {file} 文件来配置插件，打开 {link} 查看文档。"
}
```

### CODE REVIEW

File: packages/core/admin/admin/src/components/LeftMenu/index.js [hunk 1]
```diff
@@ -129,7 +134,12 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
         ) : null}
 
         {generalSectionLinks.length > 0 ? (
-          <NavSection label="General">
+          <NavSection
+            label={formatMessage({
+              id: 'app.components.LeftMenu.general',
+              defaultMessage: 'General',
+            })}
+          >
```
Comment (yangfei4913438) [inline packages/core/admin/admin/src/components/LeftMenu/index.js:142]: - react/jsx-curly-newline
> After formatting the code with Prettier, the code prompts this error

File: packages/core/admin/admin/src/content-manager/pages/ListView/FieldPicker/index.js [hunk 2]
```diff
@@ -50,7 +51,15 @@ const FieldPicker = ({ layout }) => {
...
-        customizeContent={values => `${values.length} currently selected`}
+        customizeContent={values =>
+          formatMessage(
+            {
+              id: getTrad('select.currently.selected'),
+              defaultMessage: '{count} currently selected',
+            },
+            { count: values.length }
+          )
+        }
```
Comment (yangfei4913438) [inline packages/core/admin/admin/src/content-manager/pages/ListView/FieldPicker/index.js:62]: - react/jsx-curly-newline
> After formatting the code with Prettier, the code prompts this error

File: codecov.yml [hunk 3]
```diff
@@ -6,6 +6,12 @@ comment:
 github_checks:
   annotations: false
 
+codecov:
+  status:
+    patch:
+      default:
+        target: 33%
+
```
Comment (yangfei4913438) [inline codecov.yml:14]: I just updated some translated content, but it required me to upgrade more than 10% of the codecov/patch target, which I find incredible.

Comment (yangfei4913438) [reply codecov.yml:14]: I think I need to update, at most, the unit tests related to those files I changed, and not the unit tests of many unrelated files.

File: codecov.yml [hunk 4]
```diff
@@ -6,6 +6,13 @@ comment:
 github_checks:
   annotations: false
 
+coverage:
+  status:
+    patch:
+      default:
+        target: auto
+        threshold: 33%
+
```
Comment (yangfei4913438) [inline codecov.yml:15]: The checker suggests to me that my PR is reduced by more than 10% coverage/patch, I'm not quite sure what this coverage/patch means. But I think this is not a reasonable request, I should at most only need to provide the test code changes related to my change file, other tests of unrelated code should not be written by the PR submitter.

Comment (yangfei4913438) [reply codecov.yml:15]: Maybe we can consider turning it off and just keep the codecov/project detection. For the vast majority of projects, the first thing that needs our attention is whether all the features that users need, are implemented and whether the number of bugs is less. Rather than a cold technical indicators. I also found an interesting situation when I was querying the configuration parameters of codecov/patch, there were many people searching how to lower the indicator or turn it off.

File: packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/tests/index.test.js [hunk 5]
```diff
@@ -170,28 +170,29 @@ describe('ApplicationsInfosPage || LogoInput', () => {
...
-
-      fireEvent.click(screen.getByText('Next'));
-
-      await waitFor(() =>
-        expect(
-          screen.getByText('Wrong format uploaded (accepted formats only: jpeg, jpg, png, svg).')
-        ).toBeInTheDocument()
-      );
-    });
+    // The system now supports gif format images. So this unit test is invalid.
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/tests/index.test.js:173]: the LogoInput feature does not support gif

File: packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js [hunk 6]
```diff
@@ -425,7 +425,7 @@ describe('FromComputerForm', () => {
                       <span
                         class="c8"
                       >
-                        Drag & Drop here or
+                        Click to add an asset or drag and drop one in this area
```
Comment (ronronscelestes) [inline packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js:428]: the message here should stay "Drag & Drop here or"
image

File: packages/core/upload/admin/src/components/UploadAssetDialog/tests/__snapshots__/UploadAssetDialog.test.js.snap [hunk 7]
```diff
@@ -724,7 +724,7 @@ exports[`UploadAssetDialog from computer snapshots the component 1`] = `
                               <span
                                 class="c27"
                               >
-                                Drag & Drop here or
+                                Click to add an asset or drag and drop one in this area
```
Comment (ronronscelestes) [inline packages/core/upload/admin/src/components/UploadAssetDialog/tests/__snapshots__/UploadAssetDialog.test.js.snap:727]: same here

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
hi @yangfei4913438!
first, thanks a lot for this significant contribution, lots of nice improvements in there 🙂

though I don't think I can review it the way it is now mainly because it offers many different propositions that are very different and need different reviews (and even reviewers).

I would propose to close this PR and instead open new PRs each with a defined goal:
- One for your linter/formatting propositions
- One for missing formatMessage usage + zh-Hans translation addition
- One for tests additions

File: packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/tests/index.test.js (hunk 5 above)
Comment (yangfei4913438) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/tests/index.test.js]: done

File: packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js (hunk 6 above)
Comment (yangfei4913438) [reply packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js]: done

File: packages/core/upload/admin/src/components/UploadAssetDialog/tests/__snapshots__/UploadAssetDialog.test.js.snap (hunk 7 above)
Comment (yangfei4913438) [reply packages/core/upload/admin/src/components/UploadAssetDialog/tests/__snapshots__/UploadAssetDialog.test.js.snap]: done

**Comment by yangfei4913438** [comment]
@ronronscelestes hi, I changed the content of the PR, now it's all about missing formatMessage usage + en-Hans translation addition

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
Thank you for cleaning the PR 😊
Looks good, I just added few small comments

File: examples/getstarted/src/admin/app.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 export default {
   config: {
-    locales: ['fr'],
+    locales: ['fr', 'zh-Hans'],
```
[the remark this answers is not in the document]
Comment (yangfei4913438) [reply examples/getstarted/src/admin/app.js]: done

File: packages/core/admin/admin/src/pages/ProfilePage/index.js [hunk 9]
```diff
@@ -539,7 +539,10 @@ const ProfilePage = () => {
                                     defaultMessage: '{name} mode',
                                   },
                                   {
-                                    name: upperFirst(theme),
+                                    name: formatMessage({
+                                      id: String(upperFirst(theme)).toLowerCase(),
```
[the remark this answers is not in the document]
Comment (yangfei4913438) [reply packages/core/admin/admin/src/pages/ProfilePage/index.js]: done

**Review by ronronscelestes — APPROVED** [review]
LGTM, thank you! 🌞
